src/tests/ftest/erasurecode/multiple_failure.yaml

```diff
@@ -0,0 +1,79 @@
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+    - server-C
+    - server-D
+    - server-E
+    - server-F
+  test_clients:
+    - client-A
+    - client-B
+timeout: 2000
+setup:
+  start_agents_once: False
+  start_servers_once: False
+server_config:
+  engines_per_host: 2
+  name: daos_server
+  servers:
+    0:
+      pinned_numa_node: 0
+      nr_xs_helpers: 1
+      fabric_iface: ib0
+      fabric_iface_port: 31317
+      log_file: daos_server0.log
+      bdev_class: nvme
+      bdev_list: ["aaaa:aa:aa.a"]
+      scm_class: dcpm
+      scm_list: ["/dev/pmem0"]
+      scm_mount: /mnt/daos0
+    1:
+      pinned_numa_node: 1
+      nr_xs_helpers: 1
+      fabric_iface: ib1
+      fabric_iface_port: 31417
+      log_file: daos_server1.log
+      bdev_class: nvme
+      bdev_list: ["bbbb:bb:bb.b"]
+      scm_class: dcpm
+      scm_list: ["/dev/pmem1"]
+      scm_mount: /mnt/daos1
+pool:
+    mode: 146
+    name: daos_server
+    scm_size: 25%
+    nvme_size: 93%
+    svcn: 1
+    control_method: dmg
+container:
+    type: POSIX
+    control_method: daos
+    properties: cksum:crc16,cksum_size:16384,srv_cksum:on
```

Comment:
Aren't you supposed to set rf:2? I noticed some tests killing (excluding 2 ranks). Setting rf:2 is safe. This will avoid redundancy failure (22). I am not sure how EC object class handles rf.